programs/bpf/tests/programs.rs

```diff
@@ -3524,3 +3526,120 @@ fn test_program_fees() {
     let post_balance = bank_client.get_balance(&mint_keypair.pubkey()).unwrap();
     assert_eq!(pre_balance - post_balance, expected_min_fee);
 }
+
+#[test]
+#[cfg(feature = "bpf_rust")]
+fn test_get_minimum_delegation() {
+    solana_logger::setup();
+
+    let GenesisConfigInfo {
+        genesis_config,
+        mint_keypair,
+        ..
+    } = create_genesis_config(100_123_456_789);
+    let mut bank = Bank::new_for_tests(&genesis_config);
+    bank.feature_set = Arc::new(FeatureSet::all_enabled());
+
+    let (name, id, entrypoint) = solana_bpf_loader_program!();
+    bank.add_builtin(&name, &id, entrypoint);
```

Comment:
Where are these params getting set? They should correspond to your example program right?

Reply:
I was under the impression that I was calling my example program, yes.

I've basically copied the sanity test:
https://github.com/solana-labs/solana/blob/e105547c1489b154981b75701112da86c47ccfbb/programs/bpf/tests/programs.rs#L518-L524

And I just use the string name of my program directly:
https://github.com/solana-labs/solana/blob/1ba3960af1e215d266c27576184d8abe5814d654/programs/bpf/tests/programs.rs#L3543-L3553

The logs from the test indicate that my program is getting run, but maybe I'm missing something important here?

Reply:
Ok just checking, couldn't tell from reviewing on mobile